packages/tldraw/src/lib/shapes/note/NoteShapeUtil.tsx

```diff
@@ -493,7 +496,13 @@ function getLabelSize(editor: Editor, shape: TLNoteShape) {
 
 function useNoteKeydownHandler(id: TLShapeId) {
 	const editor = useEditor()
-	const translation = useCurrentTranslation()
+	// Try to get the translation context, but fallback to ltr if it doesn't exist
+	let translation: TLUiTranslationContextType | null = null
+	try {
+		translation = useCurrentTranslation()
```

Comment:
This is somewhat unsafe due to the rules of hooks, maybe we can make this another way? Maybe just straight up using the context?